src/server/autocomplete.d

```diff
@@ -68,7 +68,9 @@ public AutocompleteResponse getDoc(const AutocompleteRequest request,
 	if (stuff.symbols.length == 0)
 		warning("Could not find symbol");
 	else foreach (symbol; stuff.symbols.filter!(a => !a.doc.empty))
-		response.docComments ~= formatComment(symbol.doc);
+		response.docComments ~= formatComment(symbol.doc) ~ r"\n";
```

Comment:
Now that I'm looking at this code again, it looks like something that should be handled by `join` or `joiner`.